xdp-filter/tests/test-xdp-filter.sh

```diff
@@ -320,6 +320,11 @@ test_python_slow()
     run_python_test test_slow
 }
 
+test_python_output()
+{
+    run_python_test test_output
```

Comment:
I'd prefer it if this test was kept in shell code; it's just a bit of output matching, should be pretty simple to do with grep; and this way the test will still work if Python is not available...